src/Command/OutputStyleAwareTrait.php

```diff
@@ -9,19 +9,18 @@
 
 trait OutputStyleAwareTrait
 {
-
     /**
      * @var OutputStyle|null
      */
-    protected $traitOutputStyle;
+    protected $output;
```

Comment:
I prefer giving the traits distinct properties so that there's less chance of annoying class composition errors. This could probably be private for that matter too.

Reply:
If we do that, we'd need to rename all the `$this->output->...` calls to `$this->traitOutputStyle->...` in all the commands...

Reply:
(which seems ugly to me: child classes shouldn't be aware of where the property comes from, a trait or a base class)

Reply:
Also: `ConsoleAwareTrait` defines `console`, not `traitConsole`

Reply:
That's fair, I agree it's ugly. The problem is we don't have either:

- A way to guarantee that `$this->output` is not null
- The ability to use PHP 8 null safe operator like `$this-output?->foo()` 

so ideally if we wanted this to be type safe we'd use `$this->getOutputStyle()` as it is guaranteed to never be null.

Reply:
For some reason the psalm github action isn't reporting any issues which has lead to this becoming an issue throughout the repo. If you run psalm locally you can see it complains about using `$this->output` because it might be null.

Reply:
So I think and appropriate approach is to split these concerns especially given the fact that we already have this problem in places in the core. So I'm going to open an issue for this instead and merge your changes because it's definitely an improvement.